### ISSUE TITLE
`Kernel32.readConsoleInputHelper` hides the underlying problem when an exception is thrown

### ISSUE THREAD

**Issue by gnodet** [fusesource/jansi#183]
See
```
Dec 11, 2020 2:24:28 PM org.jline.utils.Log logr
WARNING: Error in WindowsStreamPump
java.io.IOException: ReadConsoleInputW failed
        at org.fusesource.jansi.internal.Kernel32.readConsoleInputHelper(Kernel32.java:816)
        at org.jline.terminal.impl.jansi.win.JansiWinSysTerminal.processConsoleInput(JansiWinSysTerminal.java:134)
        at org.jline.terminal.impl.AbstractWindowsTerminal.pump(AbstractWindowsTerminal.java:460)
        at java.base/java.lang.Thread.run(Unknown Source)
```
from https://github.com/jline/jline3/issues/298#issuecomment-744618747

**Comment by bdw429s** [comment]
@gnodet Thanks for improving the logging.  What version will I need to be able to test this?

**Comment by gnodet** [comment]
This has been released in Jansi 2.1.1 today.
